soc/arm/Kconfig

```diff
@@ -57,14 +57,6 @@ config NRF_SPU_RAM_REGION_SIZE
 	  RAM region size for the NRF_SPU peripheral
 endif
 
-config CPU_HAS_FPU_DOUBLE_PRECISION
-	bool
-	depends on CPU_CORTEX_M7
```

Comment:
Do we need to be selecting this on CPU_CORTEX_M7 now?

Reply:
Not really, @galak , this was more of an extra precaution not to select the option outside Cortex-M7. But I think we can live without it.

Reply:
All the relevant Cortex-M SoCs (i.e. the ones configured with DP support) already select this symbol.

As for `depends on`, I agree with @ioannisg.